### PR TITLE
chore(docs): update linux installation instructions

### PR DESCRIPTION
**Pull request by daniellacosse**

### CODE REVIEW

File: docs/invitation_instructions.md [hunk 1]
```diff
@@ -16,29 +16,10 @@ Have an invitation to connect to an Outline server? Follow these instructions to
 | [iOS ›](https://itunes.apple.com/app/outline-app/id1356177741) | Get Outline on the App Store |
 | [Windows ›](https://raw.githubusercontent.com/Jigsaw-Code/outline-releases/master/client/Outline-Client.exe) | Download Outline.exe and double click to launch. |
```
Comment (daniellacosse) [inline docs/invitation_instructions.md:17]: @jyyi1 what are the S3 urls?

Comment (daniellacosse) [inline docs/invitation_instructions.md:17]: @jyyi1 what are the S3 urls?

Comment (fortuna) [reply docs/invitation_instructions.md:17]: Yeah, I think we should change to the S3 urls when we can.

**Review by fortuna — APPROVED** [review]

File: docs/invitation_instructions.md (hunk 1 above)
Comment (jyyi1) [reply docs/invitation_instructions.md:17]: * Windows: https://s3.amazonaws.com/outline-releases/client/Outline-Client.exe
* Linux: https://outline-releases.s3.amazonaws.com/client/Outline-Client.AppImage

**Review by jyyi1 — APPROVED** [review]

File: docs/invitation_instructions.md [hunk 2]
```diff
@@ -14,31 +14,12 @@ Have an invitation to connect to an Outline server? Follow these instructions to
 | ------------- | ------------- |
 | [Android ›](https://play.google.com/store/apps/details?id=org.outline.android.client) | If Google Play is not accessible, [get it here](https://github.com/Jigsaw-Code/outline-releases/blob/master/client/Outline.apk?raw=true). |
 | [iOS ›](https://itunes.apple.com/app/outline-app/id1356177741) | Get Outline on the App Store |
-| [Windows ›](https://raw.githubusercontent.com/Jigsaw-Code/outline-releases/master/client/Outline-Client.exe) | Download Outline.exe and double click to launch. |
+| [Windows ›](https://s3.amazonaws.com/outline-releases/client/Outline-Client.exe) | Download Outline.exe and double click to launch. |
 | [macOS ›](https://itunes.apple.com/app/outline-app/id1356178125) | Download outline.dmg, double click to install. Add Outline to your applications folder, double click to launch. |
-| Linux | See Linux set-up instructions below. |
+| [Linux ›]( https://s3.amazonaws.com/outline-releases/client/Outline-Client.AppImage) | Download Outline.AppImage and double click to launch. |
```
Comment (fortuna) [inline docs/invitation_instructions.md:19]: I just realized that this is incorrect. You can't double click to launch. You must first set the permission to executable: https://itsfoss.com/use-appimage-linux/

Comment (daniellacosse) [reply docs/invitation_instructions.md:19]: Ah, one moment.

Comment (daniellacosse) [reply docs/invitation_instructions.md:19]: https://github.com/Jigsaw-Code/outline-client/pull/1374

**Pull request merged**